symfony-nginx/usr/local/share/symfony/symfony_functions.sh

```diff
@@ -1,23 +1,26 @@
 #!/bin/bash
 
-do_symfony_build_permissions() {
+do_symfony_config_create() {
+  # Prepare a default parameters.yml. incenteev/parameters-handler can still update it
+  [ ! -f /app/app/config/parameters.yml ] && echo 'parameters: {}' > /app/app/config/parameters.yml
+}
+
+do_symfony_directory_create() {
   if [ "$SYMFONY_MAJOR_VERSION" -eq 2 ]; then
     mkdir -p /app/app/{cache,logs}
   fi
+  mkdir -p /app/app/config
   mkdir -p /app/var
+}
 
+do_symfony_build_permissions() {
   if [ "$IS_NFS" -ne 0 ]; then
-    # Prepare a default parameters.yml. incenteev/parameters-handler can still update it
-    mkdir -p /app/app/config
-    [ ! -f /app/app/config/parameters.yml ] && echo 'parameters: {}' > /app/app/config/parameters.yml
-    setfacl -R -m "d:u:$CODE_OWNER:rwX" -m "u:$CODE_OWNER:rwX" /app/
-
     # Fix permissions so the web server user can write to cache and logs folders
     if [ "$SYMFONY_MAJOR_VERSION" -eq 2 ]; then
-      setfacl -R -m "d:u:$APP_USER:rwX" -m "u:$APP_USER:rwX" /app/app/{cache,logs}
-      chmod -R ug+rw,o-rw /app/app/{cache,logs}
+      chown -R "$APP_USER:$CODE_GROUP" /app/app/{cache,logs}
+      chmod -R ug+rw,o-rwx /app/app/{cache,logs}
     fi
-    setfacl -R -m "d:u:$APP_USER:rwX" -m "u:$APP_USER:rwX" /app/var
+    chown -R "$APP_USER:$CODE_GROUP" /app/var
```

Comment:
we'll need to move this to after composer, due to composer creating files that are build:build rw-r-r